configs/stage/roles/ocm.json

```
@@ -0,0 +1,52 @@
+{
+  "roles": [
+    {
+      "name": "OCM Cluster Editor",
+      "display_name": "Cluster Editor",
+      "description": "Grants permission to edit cluster",
```

Should `cluster` be plural here like the others?
```suggestion
      "description": "Grants permission to edit clusters",
```